.infrastructure/60_cdn.tf

```diff
@@ -2,6 +2,15 @@ resource "aws_cloudfront_origin_access_identity" "main" {
   comment = "Identity to access S3 bucket."
 }
 
+## Function to manipulate the request
+resource "aws_cloudfront_function" "website_viewer_request_handler" {
+  name    = "website-viewer-request-handler"
+  runtime = "cloudfront-js-1.0"
```

Comment:
I think we could try with `cloudfront-js-2.0`, what do you think?

Reference: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudfront_function#argument-reference

Reply:
upgraded to cloudfront-js-2.0 by new commit